packages/git/src/browser/blame/blame-decorator.ts

```diff
@@ -20,10 +20,27 @@ import { GitFileBlame } from '../../common';
 import { Disposable, DisposableCollection } from '@theia/core';
 import * as moment from 'moment';
 import URI from '@theia/core/lib/common/uri';
+import { DecorationStyle } from '@theia/core/lib/browser';
 
 @injectable()
 export class BlameDecorator implements monaco.languages.HoverProvider {
 
+    constructor(
+        protected blameDecorationsStyleSheet: CSSStyleSheet | undefined = DecorationStyle.createStyleSheet('gitBlameDecorationsStyle')
+    ) {
+        new EditorDecorationStyle('git-blame-highlight', style => {
+            style.backgroundColor = 'var(--theia-gitlens-lineHighlightBackgroundColor)';
+        }, this.blameDecorationsStyleSheet);
+
+        new EditorDecorationStyle('git-blame-continuation-line::before', style => {
+            style.content = "'\u2007'"; // blank
+        }, this.blameDecorationsStyleSheet);
+
+        new EditorDecorationStyle('git-blame-continuation-line::after', style => {
+            style.content = "'\u2007'";
+        }, this.blameDecorationsStyleSheet);
+    }
```

Comment:
If you remove the `undefined` from the return options of `DecorationStyle.createStyleSheet` and `getOrCreateStyleRule`, you can shorten this constructor to this:
```typescript
    constructor(
        protected blameDecorationsStyleSheet: CSSStyleSheet = DecorationStyle.createStyleSheet('gitBlameDecorationsStyle')
    ) {
        DecorationStyle.getOrCreateStyleRule(`.${BlameDecorator.GIT_BLAME_HIGHLIGHT}`, this.blameDecorationsStyleSheet).style.backgroundColor = 'var(--theia-gitlens-lineHighlightBackgroundColor)';
        DecorationStyle.getOrCreateStyleRule(`.${BlameDecorator.GIT_BLAME_CONTINUATION_LINE}::before`, this.blameDecorationsStyleSheet).style.content = "'\u2007'"; // blank
        DecorationStyle.getOrCreateStyleRule(`.${BlameDecorator.GIT_BLAME_CONTINUATION_LINE}::after`, this.blameDecorationsStyleSheet).style.content = "'\u2007'"; // blank;
    }
```
Since we don't retain a reference to these `EditorDecorationStyle`s instances, there's no point in creating them - we might as well just insert the rules.